spacepy/pybats/bats.py

```diff
@@ -1300,29 +1418,30 @@ def gyroradius(self,velocities=('u','vth'),m_avg=3.1):
 
         if 'u' in velocities:
             try:
-                U=self['u']
+                U = self['u']
             except KeyError:
                 self.calc_utotal()
 
-        velocities_squared_sum=self[velocities[0]]**2
+        velocities_squared_sum = self[velocities[0]]**2
 
         for vname in velocities[1:]:
-            velocities_squared_sum+=self[vname]**2
+            velocities_squared_sum += self[vname]**2
 
-        v=np.sqrt(velocities_squared_sum)*1000
+        v = np.sqrt(velocities_squared_sum)*1000
 
-        if 'b' not in self: self.calc_b()
+        if 'b' not in self:
+            self.calc_b()
 
-        B=self['b']*1e-9
+        B = self['b']*1e-9
 
-        m_avg_kg=m_avg*1.6276e-27
+        m_avg_kg = m_avg*1.6276e-27
```

Comment:
The proton mass appears in a few places... should there be constants defined somewhere? A few key modules already depend on scipy, so could just use `scipy.constants`? Local definition would be fine too.